lang/cs/Org.Apache.REEF.Network/Group/Operators/Impl/BroadcastOperatorSpec.cs

```diff
@@ -35,9 +36,32 @@ public BroadcastOperatorSpec(string senderId, ICodec<T> codecType)
         {
             SenderId = senderId;
```

Comment:
You might want to call the constructor with a null PipelineDataConverter. In case any logic is in the constructor, there won't have any duplication or out of sync issue,